ctapipe/reco/hillas_intersection.py

```diff
@@ -44,14 +45,22 @@ class HillasIntersection(Reconstructor):
     for multiplicity 2 events.
     """
 
-    def __init__(self, atmosphere_profile_name="paranal"):
+    def __init__(self, atmosphere_profile_name="paranal", weighting="Konrad", config=None, parent=None, **kwargs):
```

Comment:
Make these parameters traits of the Component rather than explicit in the constructor (see other Components).  Then all you need is config, parent, and kwargs here. 

e.g. 
```py3
from ctapipe.core import traits

class HillasIntersection(Reconstructor):
    atmosphere_profile_name = traits.CaselessStrEnum(
        ['paranal',], 
        default_value="paranal", 
        help="name of atmosphere profile to use"
    ).tag(config=True)
    
    weighting = traits.CaselessStrEnum(
        ['konrad','hess'],
        default_value='konrad',
        help = 'Weighting Method name'
    ).tag(config=True)

...
...
```

Reply:
done.